src/portalnet/discovery.rs

```diff
@@ -112,8 +112,9 @@ impl Discovery {
     }
 
     /// Returns closest nodes according to given distances.
-    pub fn find_nodes_response(&self, distances: Vec<u64>) -> Vec<Enr> {
-        self.discv5.nodes_by_distance(distances)
+    pub fn find_nodes_response(&self, distances: Vec<u16>) -> Vec<Enr> {
+        let distances64: Vec<u64> = distances.iter().map(|x| (*x).into()).collect();
+        self.discv5.nodes_by_distance(distances64)
```

Comment:
If we aren't reusing this for the portal layer (which I think we don't, from the previous comment), then I think this can stay 64 bit.